moped-etl/arcgis/components_to_agol.py

```diff
@@ -247,7 +217,7 @@ def main(args):
         for feature_type in ["points", "lines", "combined", "exploded"]:
             logger.info(f"Processing {feature_type} features...")
             features_of_type = all_features[feature_type]
-            features = handle_status_updates(features_of_type)
+            features = features_of_type
```

Comment:
it is a small thing, but it would be nice to remove this unneeded reassignment here and on 241 so that the rest of the code can use `features` like in the [pre-BeautifulSoup code](https://github.com/cityofaustin/atd-moped/pull/1428/files).